packages/sanity/src/core/i18n/Translate.tsx

```diff
@@ -106,33 +109,43 @@ export function Translate(props: TranslationProps) {
   })
 
   const tokens = useMemo(() => simpleParser(translated), [translated])
-  return <>{render(tokens, props.values, props.components || {})}</>
+  const listFormat = useListFormat()
+  const formatters: FormatterFns = {
+    list: (listValues: Iterable<string>) => listFormat.format(listValues),
```

Comment:
It's very inconsequential, but the `listValues` parameter type is redundant here. It's known from the `FormatterFns` type.

```suggestion
    list: (listValues) => listFormat.format(listValues),
```